### PR TITLE
EUI-6183

### PR DESCRIPTION
**Pull request by ghost**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6183

### Change description ###
MC - 2.1.1 - Skip to main content does not skip to main content.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by RiteshHMCTS — APPROVED** [review]

**Review by SivaprakasamSivaprakasam — DISMISSED** [review]

**Review by SivaprakasamSivaprakasam — APPROVED** [review]

**Review by phillip-whitaker-hmcts — DISMISSED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by SivaprakasamSivaprakasam — APPROVED** [review]

**Review by phillip-whitaker-hmcts — APPROVED** [review]

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]
This won't work, please do not merge this change

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
    dependent on the User. -->
   <div [ngStyle]="{'background-color': backgroundColor}">
 
-    <a [routerLink]="'.'" fragment="content" class="govuk-skip-link">Skip to main content</a>
```
Comment (andywilkinshmcts) [inline src/app/components/header/header.component.html:6]: This should be something like a [routerLink]=window.location() fragment=content

File: src/app/components/header/header.component.ts [hunk 2]
```diff
@@ -27,13 +27,24 @@ export class HeaderComponent implements OnInit {
   @Input() public currentUrl: string;
   @Output() public navigate = new EventEmitter<string>();
 
+  skipLinkNavigate: string = '';
+  contentHash: string = '#content';
```
Comment (phillip-whitaker-hmcts) [inline src/app/components/header/header.component.ts:31]: need public accessor declaring

Comment (phillip-whitaker-hmcts) [inline src/app/components/header/header.component.ts:31]: need public accessor declaring

**Review by phillip-whitaker-hmcts — CHANGES_REQUESTED** [review]
couple of changes

**Review by RiteshHMCTS — APPROVED** [review]

File: src/app/components/header/header.component.ts (hunk 2 above)
Comment (cogrimes) [reply src/app/components/header/header.component.ts:31]: Will sort this guys @GaryScottHMCTS and @phillip-whitaker-hmcts

File: src/app/components/header/header.component.ts [hunk 3]
```diff
@@ -27,13 +27,24 @@ export class HeaderComponent implements OnInit {
...
+
   constructor(
     public store: Store<fromRoot.State>,
   ) {}
 
   public ngOnInit() {
   }
 
+  public checkSkipLink(urlSkipLink: string) {
```
[the remark this answers is not in the document]
Comment (cogrimes) [reply src/app/components/header/header.component.ts:40]: We definitely can. Good shout.

**Pull request merged**